File: Java/Ram.java
public class Ram extends Komponen
{
    private int kapasitasGB;
    private String ddr;

    public Ram()
    {
        this.kapasitasGB = 0;
        this.ddr = "";
    }

    public Ram(int kapasitasGB, String ddr, String merk, String nama)
    {   
        super(merk, nama);
        this.kapasitasGB = kapasitasGB;
        this.ddr = ddr;
    }

    public void set_kapasitasGB(int kapasitasGB)
    {
        this.kapasitasGB = kapasitasGB;
    }

    public void set_ddr(String ddr)
    {
        this.ddr = ddr;
    }

    public int get_kapasitasGB()
    {
        return this.kapasitasGB;
    }

    public String get_ddr()
    {
        return this.ddr;
    }
}

File: Python/Ram.py
from Komponen import Komponen

class Ram(Komponen):
    __kapasitasGB = 0
    __ddr = ""

    def __init__(self, kapasitasGB, ddr, merk, nama):
        super().__init__(merk, nama)
        self.__kapasitasGB = kapasitasGB
        self.__ddr = ddr

    def get_kapasitasGB(self):
        return self.__kapasitasGB
    
    def get_ddr(self):
        return self.__ddr
    
    def set_kapasitasGB(self, kapasitasGB):
        self.__kapasitasGB = kapasitasGB

    def set_ddr(self, ddr):
        self.__ddr = ddr